test/fixtures/cookbooks/test/recipes/config.rb

```diff
@@ -1,44 +1,62 @@
-apt_update
+if node['platform'].eql?('windows')
+  include_recipe 'chocolatey'
+  chocolatey_package 'habitat'
+  hab_package 'skylerto/splunkforwarder' do
+    version '7.0.3/20180418161444'
+  end
+
+  hab_config 'splunkforwarder.default' do
+    config(
+      directories: {
+        path: [
+          'C:/hab/pkgs/.../*.log',
+        ],
+      }
+    )
+  end
+else
+  apt_update
```

Comment:
I'm going to comment here, rather than across the rest of this fixture's changes.

We should use a platform specific suite for test kitchen, and configure .kitchen.yml, rather than make a bunch of conditional changes in the recipes. A lot of lines get changed in these recipes which make a lot of diff noise.

Reply:
You're totally right, that would absolutely be the right way. Kind of embarrassed this got committed.